### PR TITLE
Images to Users

### PR DESCRIPTION
**Pull request by jramonrod**
Added images to users, forgot to pull request here it is

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<div class="row">
```
Comment (Segebre) [inline app/views/users/new.html.erb]: Why would you add a new if it is already a modal

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,27 +1,39 @@
...
+                                <th>Usuario</th>
+                                <th>Primer Nombre</th>
+                                <th>Segundo Nombre</th>
+                                <th>Apellido</th>
+                                <th>Editar</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <td><%=link_to(image_tag(@user.get_image,:class=>"img_style"),@user) %></td>
```
Comment (Segebre) [inline app/views/users/show.html.erb:20]: Def size

**Review by Segebre — CHANGES_REQUESTED** [review]
Lo del new, ya hay un modal para new y por ende no deberia de estar esa view

File: app/views/users/show.html.erb (hunk 2 above)
Comment (jramonrod) [reply app/views/users/show.html.erb:20]: Alright I removed it, also added the add images to new user also renamed _form.erb to _form.html.erb

**Pull request closed without merging**